discord_bots/cogs/in_progress_game.py

```diff
@@ -502,14 +660,12 @@ def update_ratings(
             interaction.guild.id,
             (interaction.user.name, interaction.user.display_name),
         )
-        game_history_message: discord.Message
+        game_history_message: Message
         if config.GAME_HISTORY_CHANNEL:
-            game_history_channel: discord.abc.GuildChannel | None = (
-                interaction.guild.get_channel(config.GAME_HISTORY_CHANNEL)
+            game_history_channel: GuildChannel | None = interaction.guild.get_channel(
+                config.GAME_HISTORY_CHANNEL
             )
-            if game_history_channel and isinstance(
-                game_history_channel, discord.TextChannel
-            ):
+            if game_history_channel and isinstance(game_history_channel, TextChannel):
```

Comment:
This can be simplified to `if isinstance(game_history_channel, TextChannel):`, since `None` is a type and will fail in the `isinstance` check

Reply:
Done